ci/kmm-kmod-dockerfile.yaml

```diff
@@ -13,7 +13,8 @@ data:
 
     WORKDIR /usr/src
 
-    RUN git clone https://github.com/rh-ecosystem-edge/kernel-module-management.git
+    RUN grep super-secret-value /run/secrets/build-secret/ci-build-secret && \
+        git clone https://github.com/rh-ecosystem-edge/kernel-module-management.git
```

Comment:
```suggestion
    RUN grep super-secret-value /run/secrets/build-secret/ci-build-secret
    RUN git clone https://github.com/rh-ecosystem-edge/kernel-module-management.git
```

Reply:
I don't mind by why? I would like to think about it as git clone --auth ... so it makes sense to me to put in in the same layer.

Reply:
It can be used for pretty much anything you want, `git clone` is just an example.
I think having it in the same `RUN` as the clone makes it more confusing.